### PR TITLE
[RUNTIME-941] Add initial docs on Visualizer

### PR DESCRIPTION
**Pull request by kaustavdm**
This PR introduces a new document for Visualizers in Postman. I have added a short description, a quick idea of how the feature works, a video screencast of Visualizers in action and a Run-in-Postman button for the template demonstrated in the screencast.

This page should have more details in future, including, info on `pm.getData()` method, further explanation of the possibilities with Visualizer, and more examples. For now, let's get the ball rolling.

Reviewers, let me know if you think this is headed in the right direction and if the video snippet will render in production.

Note: this page is not linked from the left sidebar in Learning Center yet. I'll add that separately later on.

[Rendered](https://github.com/postmanlabs/postman-docs/blob/feature/RUNTIME-941-visualizer/v6/postman/sending_api_requests/visualizer.md)

[Linked Video](https://www.youtube.com/watch?v=Qj7j3QsY2ok)

### CODE REVIEW

File: v6/postman/sending_api_requests/visualizer.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+For example, your API might respond with a JSON body containing an array of objects. That data is can be better understood by putting it in a table, than looking at an array of objects. Or, you may receive a timeseries data that you can plot of a timeseries chart.
+
+## How Visualizers work
+
+Visualizers are [scripts](/docs/postman/scripts/intro_to_scripts) that you write in the Pre-request Script or Tests tab in your requests using the `pm.visualizer.set()` method.
+
+The `pm.visualizer.set()` method takes three arguments:
+
+- `layout` (required): The first argument of this method is a [Handlebars](https://handlebarsjs.com/) template string.
+- `data`: The second argument is optional layout context data that is passed to the template.
```
Comment (shamasis) [inline v6/postman/sending_api_requests/visualizer.md]: `context`?

File: v6/postman/sending_api_requests/visualizer.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- `options`: The third argument is an optional configuration object for the underlying renderer, which you can use to pass Handlebars configuration options currently.
```
Comment (shamasis) [inline v6/postman/sending_api_requests/visualizer.md]: `underlying renderer`? and `handlebars`?

File: v6/postman/sending_api_requests/visualizer.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Postman uses the information you pass to `pm.visualizer.set()` to render a HTML page in an `iframe` sandbox for Visualizer. You see this rendered HTML page inside the "Visualize" tab. The `layout` string goes into the `<body>` of this rendered page, including any JavaScript, CSS and HTML that the layout template may contain.
```
Comment (shamasis) [inline v6/postman/sending_api_requests/visualizer.md]: `an HTML`.

PS ... why is knowing about iframe and stuff needed at this stage?

File: v6/postman/sending_api_requests/visualizer.md (hunk 1 above)
Comment (shamasis) [inline v6/postman/sending_api_requests/visualizer.md]: Traditionally, we document in a recipe and build-up style... introducing everything early on does not sound homogeneous to that idea. Instead, the API ref can be appended at the end after you build it up?

Comment (kaustavdm) [reply v6/postman/sending_api_requests/visualizer.md]: Yup. See if the current write-up fits that mode better.

**Pull request merged**